DEVELOPMENT.md

```diff
@@ -221,6 +221,16 @@ with no exceptions. Particularly we don't use methods from std that throw
 exceptions but prefer the variants returning error codes (e.g. in 
 `std::filesystem`).
 
+### Namespaces
+We place all code belonging to a module named `ModuleName` or `OrbitModuleName`
+inside the top-level namespace `orbit_module_name`. We do not use nested
+namespaces.
```

Comment:
Maybe as further clarification, add something like:

"All namespaces start with the `orbit_` prefix."

Reply:
Yep, done.